src/App.js

```diff
@@ -1,25 +1,32 @@
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import Sidebar from "./components/sidebar/Sidebar";
+import "./App.css";
+import logo from "./components/sidebar/logo.png";
+import "./components/sidebar/Sidebar.css";
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+ 
+
+class App extends React.Component {
+  render() {
+    return (
+      <div className="App">
```

Comment:
Hey @ggemcila , would you like to integrate your codes in your component file to keep the App.js simpler?

Reply:
I did that now Muge!Thanks